libs/common-system/pom.xml

```diff
@@ -35,9 +36,52 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
```

Comment:
Made this it's own bundle and included it early in the platform feature file so we could remove the ~25 times we embed this module